src/components/gaza-support-section.tsx

```
@@ -0,0 +1,23 @@
+import type { GazaSupportSection } from '@/types';
+import { ColumnLayout } from './column-layout';
+
+export default function GazaSupportSection({ data }: { data: GazaSupportSection }) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'GazaSupportSection'. Consider to delete it or rename it.

'GazaSupportSection' is defined here:


(lint/suspicious/noRedeclare)